src/realm/object-store/sync/app.hpp

```diff
@@ -57,15 +57,28 @@ class App : public std::enable_shared_from_this<App>,
             public Subscribable<App> {
 public:
     struct Config {
+        // Information about the device where the app is running
+        struct DeviceInfo {
+            std::string platform;         // required, originally App::Config::platform
+            std::string platform_version; // required,  originally App::Config::platform_version
+            std::string sdk_version;      // required, originally App::Config::sdk_version
+            std::string sdk;              // required,
+            std::string cpu_arch;
+            // device_id is stored with the user configuration
```

Comment:
These comments mean there are other fields that aren't in this struct but which are sent to the server?

Reply:
Updated comments to be more clear